pkg/apis/config/types.go

```diff
@@ -47,6 +49,64 @@ type ETCD struct {
 	Backup ETCDBackup
 }
 
+type CSI struct {
+	// CSIAttacher is the configuration for the external-attacher
+	CSIAttacher *CSIAttacher
+	// CSIDriverCinder is the configuration for the csi-driver-cinder
+	CSIDriverCinder *CSIDriverCinder
+	// CSIProvisioner is the configuration for the external-provisioner
+	CSIProvisioner *CSIProvisioner
+	// CSIResizer is the configuration for the external-resizer
+	CSIResizer *CSIResizer
+	// CSISnapshotController is the configuration for the snapshot-controller
+	CSISnapshotController *CSISnapshotController
+	// CSISnapshotter is the configuration for the external-snapshotter
+	CSISnapshotter *CSISnapshotter
+	// CSILivenessProbe is the configuration for the livenessprobe
+	CSILivenessProbe *CSILivenessProbe
+}
+
+type CSIAttacher struct {
+	CSIBaseArgs
+	// RetryIntervalStart The exponential backoff for failures.
+	RetryIntervalStart *string
+	// RetryIntervalMax The exponential backoff maximum value.
+	RetryIntervalMax *string
+	// ReconcileSync Resync frequency of the attached volumes with the driver.
+	ReconcileSync *string
+}
+
+type CSIDriverCinder struct {
+	CSIBaseArgs
+}
+
+type CSILivenessProbe struct {
+	CSIBaseArgs
+}
+
+type CSIProvisioner struct {
+	CSIBaseArgs
+}
+
+type CSISnapshotter struct {
+	CSIBaseArgs
+}
+
+type CSIResizer struct {
+	CSIBaseArgs
+}
+
+type CSISnapshotController struct {
+	CSIBaseArgs
+}
```

Comment:
We don't need those structs as they just embed `CSIBaseArgs`.
Could you please use those in the `CSI` struct as suggested?
 https://github.com/gardener/gardener-extension-provider-openstack/pull/420/files#r879148760